nuxt/components/testimonial-grid.vue

```diff
@@ -1,23 +1,19 @@
 <template>
-  <section :class="containerClasses"
-    class="testimonial-list col-lg-8 row-mt-4 mx-auto justify-content-center container">
-    <component :is="'h' + headlineLevel" v-if="headline" class="testimonial-list__headline space-bottom-1">
+  <section :class="containerClasses" class="col-lg-8 row-mt-4 mx-auto justify-content-center container">
+    <component :is="'h' + headlineLevel" v-if="headline" class="space-bottom-1">
       {{ headline }}
     </component>
-    <div v-if="subline" class="testimonial-list__subline space-bottom-2">
+    <div v-if="subline" class="testimonial-grid__subline">
       {{ subline }}
     </div>
-    <div class="row row-cols-2">
-      <div v-for="(testimonial, idx) in contents.slice(0, toggleLimitValue)" :key="idx" :class="columnClass">
-        <div class="testimonial-list__content-block">
-
-          <testimonial-teaser :href="testimonial.href" :name="testimonial.name" :title="testimonial.title"
-            :img="testimonial.img" :video="testimonial.video" :bgColor="testimonial.bgColor" />
-
-        </div>
+    <div class="row">
+      <div v-if="contents" v-for="(testimonial, idx) in contents.slice(0, toggleLimitValue)" :key="idx" :class="columnClass"
```

Comment:
I would move the slice to a computed. this way you are doing slice for every render, which might not be super heavy but could be avoided and clean up the template.